spec/units/search_spec.rb

```diff
@@ -49,12 +49,13 @@
   end
 
   let :scan_response do
-    { "_scroll_id" => "abc123", "hits" => { "total" => 2 } }
+    { "_scroll_id" => "abc123", "hits" => { "total" => 2, "hits" => [
```

Comment:
I don't claim to understand what is happening here,  but why is `total` 2 but `hits` has only 1 element?

Reply:
In a real use case the results would be batched, this is simulating that.

Reply:
ah,  got it

Reply:
Does this change to the setup reveal the issue on the test (i.e. make test fails)?

If not can you add a test that reproduce the issues?

Reply:
I added a check to the persistence_spec here: https://github.com/doximity/es-elasticity/pull/50/files#diff-1b68ce8e83f431130ab26bbbaa97d4e7R151

Previously that was returning zero, before the change.  You think that's good enough or should I add more robust testing?